messages/es.json

```
@@ -0,0 +1,5 @@
+{
+  "Navbar": {
+    "menu": "Menu"
```

You need to work on your Spanish. 😆 

You can do accents on a Mac by press and holding the key and an accent selector will show up. Press and hold `u` for `ú`.

```suggestion
    "menu": "Menú"
```